tests/cases/fourslash/autoImportPackageJsonImportsLength1.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path="fourslash.ts" />
+
+// @module: nodenext
+
+// @Filename: /package.json
+//// {
+////   "imports": {
+////     "#*": "./src/*.ts"
+////   }
+//// }
+
+// @Filename: /src/a/b/c/something.ts
+//// export function something(name: string): any;
+
+// @Filename: /src/a/b/c/d.ts
+//// something/**/
+
+verify.importFixModuleSpecifiers("", ["./something"]);
```

Comment:
Usually, the codefix is supposed to return all reasonable module specifiers, while completions filters that list down to one. I think all these should be returning both the relative path and the `imports` path. (That’s probably not how the existing code for `exports` is set up because it’s _not_ reasonable to provide a relative path into `./node_modules`.)

I can definitely see some people wanting to use the `imports` path exclusively. Does the `importModuleSpecifierPreference` affect which module specifier is returned (or after both are returned, the sort order)?

Reply:
> Usually, the codefix is supposed to return all reasonable module specifiers, while completions filters that list down to one. I think all these should be returning both the relative path and the imports path. (That’s probably not how the existing code for exports is set up because it’s not reasonable to provide a relative path into ./node_modules.)

Yeah, I was thinking that would be better though that seemed like a more invasive change and currently with `compilerOptions#paths`, only a single specifier is returned and I was thinking the biggest thing here to solve is getting `imports` roughly up to par with `compilerOptions#paths` so people will practically switch to it. Do you think starting with just returning the single specifier like `compilerOptions#paths` would be okay?

> Does the importModuleSpecifierPreference affect which module specifier is returned (or after both are returned, the sort order)?

It should but I'll add some tests and fix it if it doesn't

Reply:
`importModuleSpecifierPreference: "non-relative"` now always uses `imports` (test for that in `autoImportPackageJsonImportsPreference3`)

Reply:
@andrewbranch do u still want to return all reasonable module specifiers here?

Reply:
I think that’s ideal, but I’m ok if this PR only treats `imports` like `paths`.